src/Observables.jl

```diff
@@ -49,6 +67,31 @@ end
 
 Observable(val::T) where {T} = Observable{T}(val)
 
+
+"""
+    obs = ChangeObservable(value)
```

Comment:
Also use a keyword interface?

I don't think I'd capitalize this if it isn't a type.